featuretools/tests/variables/test_variables.py

```diff
@@ -0,0 +1,29 @@
+import warnings
+
+import pandas as pd
+
+import featuretools as ft
+from featuretools.variable_types import NaturalLanguage, Text
+
+
+def test_text_depreciation():
+    data = pd.DataFrame({
+        "id": [1, 2, 3, 4, 5],
+        "text_column": ["a", "c", "b", "a", "a"],
+    })
+
+    with warnings.catch_warnings(record=True) as ws:
+        warnings.simplefilter("always")
+        es = ft.EntitySet()
+        es.entity_from_dataframe(entity_id="test", dataframe=data, index="id",
+                                 variable_types={"text_column": Text})
```

Comment:
if `Text` is replaced with `"text"` here featuretools fails to recognize the string as belonging to the `Text` class.